pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -416,12 +456,94 @@ func newDeployment(etcdstorage *samplev1alpha1.EtcdStorage) *appsv1.Deployment {
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{
 						{
-							Name:  "nginx",
-							Image: "nginx:latest",
+							Name:    "etcdproxy",
+							Image:   "quay.io/coreos/etcd:v3.2.18",
```

Comment:
followup issue to make this optionally configurable.

Reply:
Issue #10 covers this.

Reply:
Fixed in this PR.